### PR TITLE
GH-44381:adds to IMPORTANT message

### PR DESCRIPTION
**Pull request by JoeAldinger**
GH #44381

Version(s):
4.10+

Issue:
GH #44381 

Link to docs preview:
[Preview](https://deploy-preview-45594--osdocs.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-egress-firewall-ovn.html)

Additional information:

### CODE REVIEW

File: modules/nw-egressnetworkpolicy-about.adoc [hunk 1]
```diff
@@ -41,8 +41,7 @@ endif::ovn[]
 
 [IMPORTANT]
 ====
-If your egress firewall includes a deny rule for `0.0.0.0/0`, access to your {product-title} API servers is blocked.
-To ensure that pods can continue to access the {product-title} API servers, you must include the IP address range that the API servers listen on in your egress firewall rules, as in the following example:
+If your egress firewall includes a deny rule for `0.0.0.0/0`, access to your {product-title} API servers is blocked. To ensure that pods can access the {product-title} API servers, you must include the built-in join network 100.64.0.0/16 of Open Virtual Network (OVN) to allow access when using nodeports together with an EgressFirewall. You must also include the IP address range that the API servers listen on in your egress firewall rules, as in the following example:
```
Comment (Amrita42) [inline modules/nw-egressnetworkpolicy-about.adoc:44]: I think it is **node ports** in this context.

Comment (jboxman-rh) [inline modules/nw-egressnetworkpolicy-about.adoc:44]: `100.64.0.0/16`

**Review by jboxman-rh — COMMENTED** [review]
IP in code font as well.

**Pull request merged**